src/main.js

```diff
@@ -40,13 +42,14 @@ const App = Vue.extend( {
 //Define route for vue app
 //ref : http://router.vuejs.org/en/
 const router = new VueRouter( {
-	mode: 'history',
```

Comment:
Ya that's right but it causing problem while development. This is what I was telling you that it will redirect you to home directory and show error of resource not found.